src/resources/js/router/router.js

```diff
@@ -208,6 +209,7 @@ router.beforeEach((to, from, next) => {
       path: to.path
     })
     .then((res) => {
+      store.commit('badaso/SET_META', res.meta)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/andycmw/badaso/pullRequest?prid=7721006)
```suggestion
      store.commit("badaso/SET_META", res.meta)
```